.gitignore

```diff
@@ -2,6 +2,9 @@
 # Those patterns should go in a global gitignore or repository-specific excludes.
 # See http://www.gwtproject.org/makinggwtbetter.html#global_gitignore
 
-target/
-
+*/target/*
 .flattened-pom.xml
+
+# gwt caches and compiled units #
+war/gwt_bree/
```

Comment:
Where does this dir come from? I just built locally, and don't get a war dir (unless I try using gwt:test without a packaging=gwt-lib, but that looks like it needs to be its own discussion with @tbroyer separately)

Reply:
Changed to war/